components/definition/cellRules/NetPyNECellRule.js

```diff
@@ -48,7 +47,27 @@ export default class NetPyNECellRule extends React.Component {
   componentWillReceiveProps(nextProps) {
     this.setState({ currentName: nextProps.name});
   }
-
+  
+  closeImportCell = () => {
```

Comment:
Can we maybe do sth like:

```
<ImportCellParams 
        name={this.props.name}
        open={this.state.importCellOpen} 
        onRequestClose={{() =>this.setState({ importCellOpen: true })}}
/>
```
So we remove this closeImportCell and openImportCell?